src/test/java/org/apache/maven/plugins/clean/CleanMojoTest.java

```diff
@@ -175,7 +174,7 @@ public void testCleanLockedFile(CleanMojo mojo) throws Exception {
             mojo.execute();
             fail("Should fail to delete a file that is locked");
         } catch (MojoException expected) {
-            assertTrue(true);
+            assertNotNull(expected.getMessage());
```

Comment:
This is wrong too. The test should use assertThrows instead.

Reply:
I don't know if I'd call it wrong. It's simply an older style of testing these things.

Reply:
Anyway, converted to assertThrows